src/components/Icons/Icons.tsx

```diff
@@ -49,3 +55,25 @@ export const LightMode: IconType = (props) => (
 export const DarkMode: IconType = (props) => (
   <Svg path={mdiBrightness3} {...props} />
 )
+
+export const Italic: IconType = (props) => (
+  <Svg path={mdiFormatItalic} {...props} />
+)
+
+export const Bold: IconType = (props) => <Svg path={mdiFormatBold} {...props} />
+
+export const Underline: IconType = (props) => (
+  <Svg path={mdiFormatUnderline} {...props} />
+)
+
+export const TextAlignLeft: IconType = (props) => (
+  <Svg path={mdiFormatAlignLeft} {...props} />
+)
+
+export const TextAlignRight: IconType = (props) => (
+  <Svg path={mdiFormatAlignRight} {...props} />
+)
+
+export const TextAlignCenter: IconType = (props) => (
+  <Svg path={mdiFormatAlignCenter} {...props} />
+)
```

Comment:
These probably aren't icons that are part of our components - I guess they are just for the stories, so they should go in the docs section or directly in the relevant stories